CHANGELOG.md

```diff
@@ -30,3 +30,7 @@
 ### 1.1.4 (2021-07-13)
 
 * Add google analytics KPIs
+
+### 1.1.5 (2021-11-26)
+
+* update outdated github teams api call 
```

Comment:
this file was bad structured from the beginning, ideally we would like to have the latest version at the top of this file.

Can I ask you to invert the order of all of these, please?

After that, happy for this to be merged and deployed to production